### DISCUSSION TITLE
Next release schedule

### DISCUSSION THREAD

**Discussion by davidandreoletti** [RDFLib/rdflib#2240] [Development]
Hi,

Last public release (v6.2.0) was published around **July 2022** and there is now a [substantial number of commit since then](https://github.com/RDFLib/rdflib/compare/6.2.0...main). 

What's the most probable timeline for the next **2023** release ?

PS: @aucampia Could you point to the best person to ask this to please ?

**Comment by aucampia** [comment]
Hi @davidandreoletti - I was hoping to get some more changes in, but I think the most responsible thing will just be to make a new release from main, as there are no changes that can't be released.

I will make a new release by the end of this week. CC: @nicholascar @ashleysommer @white-gecko @gjhiggins

**Reply by davidandreoletti** [reply]
Superb! Thank you @aucampia

**Reply by aucampia** [reply]
Will maybe run a day or two late, I want to merge https://github.com/RDFLib/rdflib/pull/2255 and https://github.com/RDFLib/rdflib/pull/2254 and I have to finish some documentation for security and then prepare the release notes, but should be done by Monday.

**Reply by aucampia** [reply]
Security docs are ready to merge now, https://github.com/RDFLib/rdflib/pull/2270, but I still have some loose ends to tie up. I'm travelling to Sweden tomorrow for work, but I will try to finish what I can there.

**Reply by aucampia** [reply]
Release made, I will likely make 6.3.2 soon also.

**Reply by davidandreoletti** [reply]
Again, thank you @aucampia !
